cms/static/sass/elements/_system-feedback.scss

```diff
@@ -72,6 +72,60 @@
   }
 }
 
+.page-banner {
```

Comment:
@andy-armstrong: Was this needed, or is this just part of getting additional stuff into cms?